packages/shorebird_cli/pubspec.yaml

```diff
@@ -14,14 +14,16 @@ dependencies:
   checked_yaml: ^2.0.2
   cli_completion: ^0.3.0
   cli_util: ^0.4.0
+  code_push_server:
```

Comment:
Hmm.  I think we need these dependencies to work the other way (eventually at least).  It should be possible to use the public shorebird repos without shorebird_private (or other private repos), even if not all the functionality is there.

Reply:
yeah open to ideas, I'm working on getting the CI to run properly